qa/mixed-cluster/src/test/java/org/elasticsearch/backwards/IndexingIT.java

```diff
@@ -514,8 +514,13 @@ static Nodes buildNodeAndVersions(RestClient client) throws IOException {
 
     static final class Nodes extends HashMap<String, Node> {
 
+        private final String bwcNodesVersion;
```

Comment:
Now might be a good time to change this class to use composition rather than inheritance

Reply:
Oh I definitely agree, but perhaps this kind of change is better done in a separate PR? I can do it as a fast follow-up. I prefer multiple but smaller PRs, but let me know what you think about it.